Resources/Prototypes/Body/Organs/rat.yml

```diff
@@ -1,15 +1,15 @@
 - type: entity
+  parent: OrganAnimalLungs
   id: OrganRatLungs
-  parent: OrganHumanLungs
-  suffix: "rat"
+  suffix: rat
   components:
   - type: Metabolizer
     metabolizerTypes: [ Rat ]
 
 - type: entity
-  id: OrganRatStomach
   parent: OrganAnimalStomach
-  suffix: "rat"
+  id: OrganRatStomach
+  suffix: rat
```

Comment:
Would arguably be cleaner to just inherit baseorganstomach and give it Animal metabolism